fontbakery-check-ttf.py

```diff
@@ -105,6 +105,33 @@
 NAMEID_LIGHT_BACKGROUND_PALETTE = 23
 NAMEID_DARK_BACKGROUD_PALETTE = 24
 
+NAMEID_STR = {
```

Comment:
I don't understand the purpose of this dict; the strings in the keys and value are so similar, why not just use the strings in the keys directly?